### DISCUSSION TITLE
Annoying message nagging too much about updating server

### DISCUSSION THREAD

**Discussion by delyan** [jellyfin/jellyfin-androidtv#4235] [General]
Do I miss something important, or it is just me getting annoyed every month the app nagging about updating Jellyfin server if wanting to continue using the app? Why so? While I can do that, its getting annoying after few months.

**Comment by nielsvanvelzen** [comment]
I think it's just you. We only use this message when a new release of the app is coming to warn users so they can update their server in time. Otherwise once the app update releases you won't be able to watch anything. This only happens about 2 times per year with the current release cycle. It's up to you to update or not to update.

**Reply by delyan** [reply]
Yes, it is up to me of course, but the point is - what's changing so much at the server, so that the app will stop play anything? Is it protocols changing, something else? Because the message is quite intrusive, it is easy for somebody to conclude things are quite experimental and turns them off Jellyfin.

**Reply by nielsvanvelzen** [reply]
Look at the server changelogs to find out what changed. I'm updating the app to a newer API version when this warning is shown, and can no longer support older server versions unless I make maintaining this app my full time job.

**Reply by delyan** [reply]
In any case, thanks Niels for what you're doing, it is really appreciated!
